src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -142,15 +120,33 @@ def assertCbsdRecord(self, registration_request, grant_request, grant_response,
           self.assertEqualToDeviceOrPreloadedConditionalParam('indoorDeployment', \
               device['installationParam'], reg_conditional_device_data['installationParam'],\
                 cbsd_record[0]['installationParam'])
-          #  parameters should equal to device or to default value            
-          self.assertEqualToDeviceOrPreloadedConditionalParamOrDefaultValue('antennaAzimuth', \
-              device['installationParam'], reg_conditional_device_data['installationParam'],\
-                cbsd_record[0]['installationParam'], 0)
-                
-          self.assertEqualToDeviceOrPreloadedConditionalParamOrDefaultValue('antennaBeamwidth', \
-              device['installationParam'], reg_conditional_device_data['installationParam'],\
-                cbsd_record[0]['installationParam'], 360)
-            
+          #  parameters should equal to device or to default value
+          registered_antenna_azimuth = device['installationParam']['antennaAzimuth'] \
+            if 'antennaAzimuth' in device['installationParam'] \
+            else reg_conditional_device_data['installationParam']['antennaAzimuth'] \
+            if ['antennaAzimuth'] in reg_conditional_device_data['installationParam'] else None
```

Comment:
I think you have an extra pair of `[]` (here and line 131).

Reply:
done, thank yo